server/routes/journeys/develop-an-initial-plan/identified-needs/view.njk

```diff
@@ -0,0 +1,160 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% extends "partials/layout.njk" %}
+
+{% set fullWidth = identifiedNeeds.length %}
+{% set pageTitle = "Identified Need - Develop an initial plan" %}
+
+{% block innerContent %}
+    <span class="govuk-caption-l">Develop an initial plan</span>
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-7">Identified needs for {{ (prisoner.firstName + " " + prisoner.lastName) | convertToTitleCase }}</h1>
+
```

Comment:
we have a filter inherited/stolen from the ccrd library to do `prisoner | firstNameSpaceLastName` that is more comprehensive than convertToTitleCase